src/pudl/package_data/settings/etl_full.yml

```diff
@@ -1,134 +1,159 @@
 ferc1_to_sqlite_settings:
-###########################################################################
-# FERC FORM 1 DB CLONE SETTINGS
-###########################################################################
-    # What years of original FERC data should be cloned into the SQLite DB?
-    years: [
-        1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003,
-        2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013,
-        2014, 2015, 2016, 2017, 2018, 2019, 2020
```

Comment:
I really like having these big lists of years organized in an array that's 10 years across so you can more easily see if there's a year missing / duplicated based on the last digits. And it's much more compact.

Reply:
My bad. My vscode formatter disagrees with PUDL's formatting.